src/site/content/en/blog/vitals-tools/index.md

```diff
@@ -1,184 +1,347 @@
+{% endAside %}
+
+Lighthouse runs simulate a mid-tier mobile device on a throttled slow 4G connection. This may find issues that wouldn't ordinarily appear on high-speed devices or fast internet connections.
 
-Once you identify a type of page that has Core Web Vitals related issues, you can use PageSpeed Insights to learn about specific optimization suggestions for representative pages.
+Even so, this simulated throttling may not be representative of the variety of user experiences among your website's user base at the 75th percentile. However, these metrics are an indicator of where performance problems exist, and may translate into better performance overall in the field if the problems Lighthouse finds are addressed.
 
-#### web.dev
+#### Analyze in real time with the Web Vitals extension
 
-[web.dev/measure](/measure/) allows you to measure the performance of your page over time, providing a prioritized list of guides and codelabs on how to improve. It's measurement is powered by PageSpeed Insights. The measure tool now also supports the Core Web Vitals metrics, as shown below:
+The Web Vitals Chrome extension shows Core Web Vitals in real time during startup _and_ while browsing a page. Because of this, it can capture FID, as well as layout shifts that occur after startup.
 
 <figure class="w-figure">
-  {% Img src="image/admin/ryoV1T1PhxUmo9zdCsDe.png", alt="Measure Core Web Vitals metrics over time and get prioritized guidance with the web.dev measure tool", width="800", height="459", class="w-screenshot" %}
+  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/FfyQpEEpJYkjW39eDz4S.png", alt="A screenshot of the Web Vitals Chrome extension panel. It shows the state of each Core Web Vital, and how the current page experience compares with the 75th percentile of end user experiences as gathered by CrUX (if available).", width="800", height="556" %}
 </figure>
 
-### Web Vitals extension
+It's best to think of the Web Vitals extension as a spot-checking tool to find performance issues, not a comprehensive debugging tool&mdash;that's a job for the performance tab in Chrome's DevTools.
+
+#### Drill down with the performance tab
 
-The Web Vitals extension measures the three Core Web Vitals metrics in real-time for (desktop) Google Chrome. This is helpful for catching issues early on during your development workflow and as a diagnostic tool to assess performance of Core Web Vitals as you browse the web.
+The performance tab in Chrome's DevTools profiles all page behavior during a recorded period of time. When populated, a layer in the timeline labeled "experience" appears, which highlights layout shifts and the elements responsible for those shifts.
```

Comment:
s/and the elements responsible for those shifts/and the elements that shifted/

The "elements responsible for those shifts" could be interpreted as the elements that _caused_ the shifts, and that's not something that DevTools shows.